cmd/kata-manager/kata-manager.sh

```diff
@@ -300,10 +300,21 @@ get_git_repo()
 	git clone "$repo_url" "$local_dest" || (rm -fr "$local_dest" && exit 1)
 }
 
+set_kata_release()
+{
+	local -r install_script="$1"
+	local -r release="$2"
+
+	check_file "$install_script"
+
+	sudo sed -i "s/master/${release}/g" "$install_script"
```

Comment:
Q. why don't use the `BRANCH` variable instead?

afaik all scripts have something like this `BRANCH="${BRANCH:-master}"`

Reply:
I updated it to pass the BRANCH variable to the script without modifying it